post-til-private/src/main/java/no/difi/meldingsutveksling/dpi/DpiReceiptStatus.java

```diff
@@ -34,20 +35,23 @@
         mapper.put(Feil.class, FEIL);
     }
 
-    final String description;
+    final String status;
     private final BiConsumer<String, LogstashMarker> loggerMethod;
 
-    ReceiptType(String description, BiConsumer<String, LogstashMarker> loggerMethod) {
-        this.description = description;
+    DpiReceiptStatus(String description, BiConsumer<String, LogstashMarker> loggerMethod) {
+        this.status = description;
         this.loggerMethod = loggerMethod;
     }
 
+    public String getStatus() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1161)